.gitignore

```diff
@@ -1,4 +1,5 @@
 venv
+.idea
```

Comment:
I don't see the need to add the settings folder, because we will have to add the rest of the editors 😅, if you don't want it to bother you, add `.gitignore` inside it and put it in it

Reply:
You can also add [global gitignore](https://stackoverflow.com/a/22885996)